src/task/CreatePeerConnectionTask.ts

```diff
@@ -187,7 +226,13 @@ export default class CreatePeerConnectionTask extends BaseTask implements Remova
       width = cap.width as number;
       height = cap.height as number;
     }
-    tile.bindVideoStream(attendeeId, false, stream, width, height, streamId);
+    const externalUserId = this.context.realtimeController.realtimeExternalUserIdFromAttendeeId(
+      attendeeId
+    );
+    tile.bindVideoStream(attendeeId, false, stream, width, height, streamId, externalUserId);
```

Comment:
If `externalUserId` is `null` won't this cause the video tile update to not include a non-empty externalUserId? Isn't the goal to ensure that there is one when the tile is bound so that callbacks always have the externalUserId?

Reply:
if ecternalUserId is null, binding will take place twice

Reply:
```
if (externalUserId) {
  tile.bindVideoStream(attendeeId, false, stream, width, height, streamId, externalUserId);
} else {
  this.bindExternalIdToRemoteTile(tile, attendeeId, stream, width, height, streamId);
}
```

Reply:
@RidipDe still need to implement above logic ^